docs/tutorial.rst

```diff
@@ -127,7 +125,7 @@ Create a template called ``genres.html`` in your template directory and put this
 
     {% load mptt_tags %}
     <ul>
-        {% recursetree nodes %}
+        {% recursetree genres %}
```

Comment:
Renaming `nodes` to `genres` is fine, but you should also do this on the new line 113. Also, please add a single space after the `:` when creating the context dictionary while you're at it :)

Thanks!

Reply:
Applied! :)